modules/cloudfront_distribution/cloudfront-distribution.yaml

```diff
@@ -94,6 +94,11 @@ inputs:
     validator: str(required=False)
     description: ID of Route53 hosted zone to add a record for. By default uses the one created by the DNS module if the module is found.
     default: ""
+  - name: forward_https
+    user_facing: true
+    validator: bool(required=False)
+    description: Should cloudformation forward to https port 443 instead of http port 80?
```

Comment:
So the user needs to set this to true if they are using the dns module?

Reply:
They don't _need_ to set it to true, they can set it to true if their lb has a valid ssl cert (either by using the one opta created by setting delegated true, or one they create themselves)